### PR TITLE
Update iban_rules.rb

### PR DESCRIPTION
**Pull request by prokopsimek**
because I want to read yaml without any modification of regexp as is on line 23

### CODE REVIEW

**Comment by prokopsimek** [comment]
@iulianu pls merge :)

File: lib/iban-tools/iban_rules.rb [hunk 1]
```diff
@@ -15,15 +15,23 @@ def [](key)
...
     def self.defaults
-      load_from_string( File.read(File.dirname(__FILE__) + "/rules.yml") )
+      load_from_string
     end
 
-    def self.load_from_string( string )
-      rule_hash = YAML.load(string)
-      rule_hash.each do |country_code, specs|
+    def self.load_from_string
+      hash_copy = rule_hash
```
Comment (sunny) [inline lib/iban-tools/iban_rules.rb:22]: This doesn’t actually create a copy. 🤔

What seems to be the issue you are trying to fix :?

Comment (prokopsimek) [reply lib/iban-tools/iban_rules.rb:22]: lol, I really don't know after 7 years...

Comment (sunny) [reply lib/iban-tools/iban_rules.rb:22]: Haha, I think it can be closed then 😅

Comment (prokopsimek) [reply lib/iban-tools/iban_rules.rb:22]: y

**Pull request closed without merging**